app/assets/javascripts/discourse/tests/acceptance/category-new-test.js

```diff
@@ -117,7 +117,7 @@ acceptance("New category preview", function (needs) {
 
     let previewBadgeColor = document
       .querySelector(".category-color-editor .badge-category")
-      .style.getPropertyValue("--category-badge-color")
+      .style.getPropertyValue("--category-color")
```

Comment:
btw. no plugins/themes rely on the previous vars?

Reply:
I already had two PRs open for the themes that use the previous name, but then I realized there might be more themes we do not know about, so I gave up on the idea to rename these vars.